test/jdk/java/lang/invoke/indify/Indify.java

```diff
@@ -168,57 +268,65 @@ public void parseOptions(List<String> av) throws IOException {
                 }
                 switch (a) {
                 case "--java":
-                    return;  // keep this argument
+                    return;
                 case "-d": case "--dest": case "-d=": case "--dest=":
                     dest = new File(a2 != null ? a2 : maybeExpandProperties(av.remove(1)));
                     break;
                 case "-cp": case "--classpath":
                     classpath = maybeExpandProperties(av.remove(1)).split("["+File.pathSeparatorChar+"]");
                     break;
                 case "-k": case "--keepgoing": case "--keepgoing=":
-                    keepgoing = booleanOption(a2);  // print errors but keep going
+                    keepgoing = booleanOption(a2);
                     break;
                 case "--expand-properties": case "--expand-properties=":
-                    expandProperties = booleanOption(a2);  // expand property references in subsequent arguments
+                    expandProperties = booleanOption(a2);
                     break;
                 case "--verify-specifier-count": case "--verify-specifier-count=":
-                    verifySpecifierCount = Integer.valueOf(a2);
+                        assert a2 != null;
+                        verifySpecifierCount = Integer.parseInt(a2);
                     break;
                 case "--overwrite": case "--overwrite=":
-                    overwrite = booleanOption(a2);  // overwrite output files
+                    overwrite = booleanOption(a2);
                     break;
                 case "--all": case "--all=":
-                    all = booleanOption(a2);  // copy all classes, even if no patterns
+                    all = booleanOption(a2);
                     break;
                 case "-q": case "--quiet": case "--quiet=":
-                    quiet = booleanOption(a2);  // less output
+                    quiet = booleanOption(a2);
                     break;
                 case "-v": case "--verbose": case "--verbose=":
-                    verbose = booleanOption(a2);  // more output
```

Comment:
Why the comments have been removed?

Reply:
Detailed explanations are above in those options variables declaration.
Comments returned